GeneratorParam/ExodusDecayer.cxx

```diff
@@ -127,14 +132,15 @@ void ExodusDecayer::Init()
     delta_omega      = (omasspion / omegamass) * (omasspion / omegamass);
     delta_etaprime   = (omassgamma / etaprimemass) * (omassgamma / etaprimemass);
     delta_etaprime_toOmega   = (omegamass / etaprimemass) * (omegamass / etaprimemass);
-    delta_phi        = (omasseta / phimass) * (omasseta / phimass);    
+    if (fDecayToDimuon == 0) delta_phi = (omasseta / phimass) * (omasseta / phimass);    
+    else delta_phi = (omassgamma / phimass) * (omassgamma / phimass);    
     delta_phi_toPi0        = (pionmass / phimass) * (pionmass / phimass);    
 
     // create pair mass histograms for Dalitz decays of pi0, eta, omega, eta' and phi
     fEPMassPion = new TH1F("fEPMassPion", "Dalitz electron pair from pion", nbins, min, maxpion);
     fEPMassPion->SetDirectory(0);
-    fEPMassEta = new TH1F("fEPMassEta", "Dalitz electron pair from eta", nbins, min, maxeta);
-    fEPMassEta->SetDirectory(0);
+    fEPMassEtaDalitz = new TH1F("fEPMassEta", "Dalitz electron pair from eta", nbins, min, maxeta);
```

Comment:
Just a tiny comment. For consistency maybe name this histogram also "fEPMassEtaDalitz"
```suggestion
    fEPMassEtaDalitz = new TH1F("fEPMassEtaDalitz", "Dalitz electron pair from eta", nbins, min, maxeta);
```